### PR TITLE
Add fixit

### PR DESCRIPTION
**Pull request by apmorton**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by apmorton** [comment]
@conda-forge/help-python, ready for review!

(windows failure appears systemic)

File: recipes/fixit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python
+    - click >=8.0
+    - libcst >=0.3.18
+    - moreorless >=0.4.0
+    - packaging >=21
+    - tomli >=2.0  # [py<311]
```
Comment (ocefpaf) [inline recipes/fixit/meta.yaml:31]: It can probably be noarch if we are OK installing `tomli` in >=311. What do you think?

Comment (apmorton) [reply recipes/fixit/meta.yaml:31]: I originally did that, but decided against it because it is a genuine unused dependency.

(as opposed to "try using tomli if there, otherwise use tomllib").

Comment (ocefpaf) [reply recipes/fixit/meta.yaml:31]: We are between an unused dependency vs several artifacts that could be one and save CI time. In conda-forge we are usually biased to avoid the latter.

Comment (apmorton) [reply recipes/fixit/meta.yaml:31]: If you insist, I will change this - but this policy does seem at-odds with the defaults in grayskull, which states:

> The main goal of this project is to generate concise recipes for conda-forge

Comment (ocefpaf) [reply recipes/fixit/meta.yaml]: Sure. Grayskull is one thing, conda-forge is another. Also, note that grayskull has a conda-forge strict option that tries to make it closer to our latest policies, it may be outdated though.

Comment (apmorton) [reply recipes/fixit/meta.yaml]: I generated this recipe using `mamba greyskull pypi --strict-conda-forge fixit`

Comment (ocefpaf) [reply recipes/fixit/meta.yaml]: Yep, that is the option but, as I mentioned above, it is probably a bit outdated. It is hard to capture the latest policies. Well, we don't even write everything down b/c they change to quickly :-)

TL;DR ideally we should have a semi-arch for Python like we have for OS-noarch now.
